css/sprint.css

```diff
@@ -310,6 +312,13 @@ a:hover .partner-attendee {
     background-color: #fce6ca;
 }
 
+.resources {
+    background-color: #fff;
+}
+.resources .sprint-columns {
+    padding: 5em 10em;
```

Comment:
The spacing between the title and the links actually looks a wee bit big to me...